pkg/envoyfilters/envoyfilters.go

```diff
@@ -52,6 +57,35 @@ func BuildAPIEnvoyFilterSpecForHelmChart(
 	}, nil
 }
 
+// BuildIngressEnvoyFilterSpecForHelmChart assembles EnvoyFilter patches for
+// endpoints using the seed ingress domain.
+func BuildIngressEnvoyFilterSpecForHelmChart(
+	cluster *controller.Cluster, rule *ACLRule, alwaysAllowedCIDRs []string, cfg map[string]interface{},
+) error {
+	seedIngressDomain := helper.GetSeedIngressDomain(cluster.Seed)
+	if seedIngressDomain != "" {
```

Comment:
If there is no seed ingress domain, the spec of the corresponding `EnvoyFilter` is empty. Is this still valid? Should we rather drop the `EnvoyFilter` in this case?